### DISCUSSION TITLE
How to get success and error at  different input after submit?

### DISCUSSION THREAD

**Discussion by amantiwari1** [react-hook-form/react-hook-form#4709] [Q&A]
it is working to get error from formstate and show error in the frontend is fine
but in success no luck to show success in any input which is already correct while other inputs able to error
any idea?

**Comment by bluebill1049** [comment]
need more context and example.

**Comment by amantiwari1** [comment]
for example
![image](https://user-images.githubusercontent.com/53963065/114186118-18fdd600-9964-11eb-9827-d34ed244b1b7.png)

above pic, i managed to able to show 2 errors in red color while the other can't manage 2 success in react hook form and there is no one example in your repo

it should be allow success and error at the same time
